javascript-source/systems/auctionSystem.js

```diff
@@ -58,22 +74,34 @@
             auction.timer = parseInt(timer);
         }
 
-        auction.status = true;
+        auction.isActive = true;
+
 
-        $.say($.lang.get('auctionsystem.opened', $.getPointsString(increments), $.getPointsString(minimum)));
+        $.say($.lang.get('auctionsystem.opened', ((auction.usePoints) ? $.getPointsString(increments): increments), ((auction.usePoints) ? $.getPointsString(minimum) : minimum)));
 
         if (timer > 0) {
             $.say($.lang.get('auctionsystem.auto.timer.msg', timer));
-            a = setTimeout(function() {
-                warnAuction(true);
-                clearInterval(a);
-            }, (timer / 2) * 1000);
-            b = setTimeout(function() {
-                closeAuction();
+            if (timer > 30) {
+                a = setTimeout(function() { //Warn once 30 seconds before the end
+                    warnAuction(true, undefined, 30);
```

Comment:
Codacy has a fix for the issue: [Replace `····················` with `··········`](https://app.codacy.com/gh/Psychoboy/PhantomBot/pullRequest?prid=9180768)
```suggestion
          warnAuction(true, undefined, 30);
```